specification/maps/data-plane/Render/preview/2.0/render.json

```diff
@@ -138,15 +138,15 @@
           },
           {
             "value": "microsoft.imagery",
-            "description": "A combination of satellite and aerial imagery. Only available in S1 pricing SKU.<br>\n\nSupports zoom levels 1 through 19. Format: raster (jpeg)."
+            "description": "A combination of satellite or aerial imagery. Only available in S1 and G2 pricing SKU.<br>\n\nSupports zoom levels 1 through 19. Format: raster (png)."
           },
           {
             "value": "microsoft.weather.radar.main",
             "description": "Weather radar tiles. Latest weather radar images including areas of rain, snow, ice and mixed conditions. Please see [coverage information](https://aka.ms/AzureMapsWeatherCoverage) for Azure Maps Weather service.  To learn more about the Radar data, please see [Weather concepts](https://aka.ms/AzureMapsWeatherConcepts).<br>\n\nSupports zoom levels 0 through 15. Format: raster (png)."
           },
           {
             "value": "microsoft.weather.infrared.main",
-            "description": "Weather infrared tiles. Latest Infrared Satellite images shows clouds by their temperature.  Please see [coverage information](https://aka.ms/AzureMapsWeatherCoverage) for Azure Maps Weather service. To learn more about the returned Satellite data, please see [Weather concepts](https://aka.ms/AzureMapsWeatherConcepts).<br>\n\nSupports zoom levels 0 through 15. Format: raster (png)."
+            "description": "Weather infrared tiles. Latest Infrared Satellite images shows clouds by their temperature.  Please see [coverage information](https://aka.ms/AzureMapsWeatherCoverage) for Azure Maps Weather service. To learn more about the returned Satellite data, please see [Weather concepts](https://learn.microsoft.com/en-us/azure/azure-maps/weather-services-concepts#satellite-images).<br>\n\nSupports zoom levels 0 through 15. Format: raster (png)."
```

Comment:
Do not include locale in doc (`en-us` here). Your user may not be all American...